cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -80,15 +82,40 @@ func DeployedDNSAddon(client clientset.Interface) (kubeadmapi.DNSAddOnType, stri
 	}
 }
 
+// deployedDNSReplicas returns the replica count for the current DNS deployment
+func deployedDNSReplicas(client clientset.Interface, replicas int32) (*int32, error) {
+	deploymentsClient := client.AppsV1().Deployments(metav1.NamespaceSystem)
+	deployments, err := deploymentsClient.List(metav1.ListOptions{LabelSelector: "k8s-app=kube-dns"})
+	if err != nil {
+		return &replicas, errors.Wrap(err, "couldn't retrieve DNS addon deployments")
+	}
+	switch len(deployments.Items) {
+	case 0:
+		return &replicas, errors.New("no DNS addons found")
```

Comment:
Let's remove the error from here, just return nil (so that we don't break `kubeadm init`).

Reply:
It was a `errors.Wrap(err, "no DNS addons found")` before on the previous version of the PR, but a `nil` is certainly better in this case, since `err` would always be `nil` here.

Reply:
whoops, my local had it. Thank you!